models/wrf_hydro/README.rst

```diff
@@ -0,0 +1,664 @@
+################
+WRF-Hydro README
+################
+
+Contents
+========
+
+#. `Overview`_
+#. `Description of this Directory within the DART Repository`_
+#. `To Setup an Experiment`_
+#. `Description of External Directories on GLADE`_
+#. `For the Impatient - Skip the Fundamentals and Get Frustrated`_
+#. `Observations`_
+#. `Namelist`_
+#. `Input Files`_
+#. `Output Files`_
+#. `Terms of Use`_
+
+Overview
+========
+
+
+
+
+
+The `NOAH <http://www.ral.ucar.edu/research/land/technology/lsm.php>`_ **Land
+Surface Model** and **Data Assimilation Research Testbed (DART)** may now be
+used for assimilation experiments.
```

Comment:
Yeah ... I'd like to replace `models/noah/README.rst` with this entire file.
At some point, we will update the content of this file to be more specific to WRF-Hydro (the channel model configuration)